### DISCUSSION TITLE
Fill Triangles

### DISCUSSION THREAD

**Discussion by IlyaTaidi** [matteobruni/tsparticles#1838] [Q&A] [answered]
Hey Matteo,

Sorry to keep filling you discussion forum so much but I have one last question.

While the other issues get solved, I downgraded back to the original particle.js. That being said, I absolutely love the fill triangles option you have in this repo. Is there a way to achieve this same, or similar affect to the og particle.js?

Thanks in advance.

**Comment by matteobruni** [comment, marked as answer]
Sorry but I'm not giving support on other libraries. You can try tsParticles `1.18` version, as you can see [here](https://codepen.io/matteobruni/pen/WNQgXJZ) link triangles are working fine, maybe something were broken in one of the updates and I'm figuring out what

**Reply by matteobruni** [reply]
I think I fixed it in the upcoming `1.32.0`, it's not ready to be released yet but it will be in some days I hope.
